lib/synth-audio.js

```diff
@@ -95,8 +95,9 @@ async function synthAudio(client, logger, stats, {
   } else  if (vendor.startsWith('custom')) {
     assert.ok(credentials.custom_tts_url, `synthAudio requires custom_tts_url in credentials when ${vendor} is used`);
   }
-
+  const account_sid = credentials.account_sid;
```

Comment:
oh hmm.  I did not think of this.  I don't think account_sid will be part of credentials, will it ?

Reply:
you are right. in https://github.com/jambonz/jambonz-feature-server/blob/main/lib/session/call-session.js#L585

the account_sid is not there, I will introduce new parameter to the function

Reply:
fixed